src/tests/ftest/security/cont_owner.yaml

```diff
@@ -0,0 +1,22 @@
+hosts:
+  test_servers: 1
+  test_clients: 1
+timeout: 1200
+server_config:
+  name: daos_server
+  port: 10001
+  engines_per_host: 1
+  engines:
+    0:
+      targets: 4
+      nr_xs_helpers: 0
+      storage:
+        0:
+          class: ram
+          scm_mount: /mnt/daos
+          scm_size: 4
+  system_ram_reserved: 1
+pool:
+  scm_size: 138374182
```

Comment:
Where does this number come from? Could we use a simple human-readable size?

Reply:
Copied from an existing test yaml file. Do we actually have to specify this, or is there a default? There's no particular reason to use a specific size.

Reply:
Fixed to use a human readable size.

Reply:
No, there isn't a default pool size in the framework. Thanks!